### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by barnjamin**

### CODE REVIEW

File: connect/package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "build": "npm run build:cjs && npm run build:esm",
     "rebuild": "npm run clean && npm run build",
     "clean": "rm -rf ./dist && rm -rf ./.turbo",
-    "lint": "npm run prettier && eslint --fix",
+    "lint": "npm run prettier && eslint --fix ./src --ext .ts",
```
Comment (artursapek) [inline connect/package.json:41]: can you put these settings (`./src --ext .ts`) into `.eslintrc.json`?

Comment (artursapek) [reply connect/package.json:41]: (and same for all the other instances of this)

Comment (artursapek) [reply connect/package.json:41]: also, can we have a single top-level `.eslintrc.json` that each package uses? `../.eslintrc.json` etc

Comment (barnjamin) [reply connect/package.json:41]: they all use the same eslint, will see if i can add this to the config tho

Comment (barnjamin) [reply connect/package.json:41]: full 5 min of looking, didn't see how, tomorrows ben problems

Comment (artursapek) [reply connect/package.json:41]: apparently its actually not possible so nvm

**Pull request merged**